ERCS/erc-0000.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 0000
+title: EIP-712 Extensions for Account Abstraction
+description: Improvements for EIP-712 to support smart contract accounts.
+author: Francisco Giordano (@frangio)
+discussions-to: https://ethereum-magicians.org/t/eip-eip-712-extensions-for-account-abstraction/21436
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7803-eip-712-extensions-for-account-abstraction/21436
```
Changed category, added ERC number on Eth Magicians.  Can you update Eth Magicians topic to link to this PR.